lab-kyle/lib/response.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Excellent!  This is a good module to abstract some of the route response functionality.